P_rect_2: [
  7.215377000000e+02, 0.000000000000e+00, 6.095593000000e+02, 4.485728000000e+01, 
  0.000000000000e+00, 7.215377000000e+02, 1.728540000000e+02, 2.163791000000e-01,
  0.000000000000e+00, 0.000000000000e+00, 1.000000000000e+00, 2.745884000000e-03]
R_rect_0: [
  9.999239000000e-01, 9.837760000000e-03, -7.445048000000e-03, 0.000000000000e+00, 
  -9.869795000000e-03, 9.999421000000e-01, -4.278459000000e-03, 0.000000000000e+00,
  7.402527000000e-03, 4.351614000000e-03, 9.999631000000e-01, 0.000000000000e+00, 
  0.000000000000e+00, 0.000000000000e+00, 0.000000000000e+00, 1.000000000000e+00]
Tr_velo_to_cam: [
  7.533745000000e-03, -9.999714000000e-01, -6.166020000000e-04, -4.069766000000e-03, 
  1.480249000000e-02, 7.280733000000e-04, -9.998902000000e-01, -7.631618000000e-02,
  9.998621000000e-01, 7.523790000000e-03, 1.480755000000e-02, -2.717806000000e-01, 
  0.000000000000e+00, 0.000000000000e+00, 0.000000000000e+00, 1.000000000000e+00]




